src/main.c

```diff
@@ -136,7 +136,7 @@ dump_conntrack(GHashTable *ips_to_migrate)
 {
     struct nfct_handle *handle;
 
```

Comment:
add a comment describing what we discussed?

Reply:
Done